### PR TITLE
TtDileptonAnalyzer migration to dileptonCommon

### PR DESCRIPTION
**Pull request by jhgoh**
This PR makes TtDileptonAnalyzer to inherit from the dileptonCommon and clear up duplicated codes.
Since there's no custom action in the TtDileptonAnalyzer, the analyzer becomes simple wrapper to the dileptonCommon, but there can be other extensions in the future.

@kuyoun , I found there's something strange in the analyzer code to save kinsolver results from parton level information.

### CODE REVIEW

File: CatAnalyzer/plugins/dileptonCommon.cc [hunk 1]
```diff
@@ -198,16 +211,16 @@ void dileptonCommon::setBranchCommon(TTree* tr, int sys) {
...
-  tr->Branch("partonjet1", "TLorentzVector", &b_pseudojet1);
-  tr->Branch("partonjet1_CSVInclV2", &b_pseudojet1_CSVInclV2, "partonjet1_CSVInclV2/F");
-  tr->Branch("partonjet2", "TLorentzVector", &b_pseudojet2);
-  tr->Branch("partonjet2_CSVInclV2", &b_pseudojet2_CSVInclV2, "partonjet2_CSVInclV2/F");
-  tr->Branch("partontop1", "TLorentzVector", &b_pseudotop1);
-  tr->Branch("partontop2", "TLorentzVector", &b_pseudotop2);
-  tr->Branch("partonnu1", "TLorentzVector", &b_pseudonu1);
-  tr->Branch("partonnu2", "TLorentzVector", &b_pseudonu2);
-  tr->Branch("partonttbar", "TLorentzVector", &b_pseudottbar);
-  tr->Branch("partonttbar_dphi", &b_pseudottbar_dphi, "partonttbar_dphi/F");
```
Comment (jhgoh) [inline CatAnalyzer/plugins/dileptonCommon.cc:210]: Here "parton" were pointing to "b_pseudo". So the partonjet and pseudojet are pointing to the same object, results from the pseudotop-tuned kinematic reconstruction, up to event selection and random number effect.

**Comment by geonmo** [comment]
@jhgoh "using namespace" 들을 지우면 더 복잡하게 보이지 않을까요?

**Comment by geonmo** [comment]
@jhgoh 제가 dileptonCommon 을 좀 고친게 있는데 먼저 머지하시면 제가 수정 후 다시 올릴게요.

**Comment by jhgoh** [comment]
header에 using namespace가 있으면 안돼요.

**Comment by jhgoh** [comment]
건모씨 먼저 올리시고 나서 이걸 적용할지 말지 보는 쪽으로 할께요. 테크니컬한 변경이라서요.

**Pull request merged**